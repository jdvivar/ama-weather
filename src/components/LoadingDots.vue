<template>
  <span class="loading-dots">
    <span>.</span><span>.</span><span>.</span>
  </span>
</template>

<script>
export default {
  name: 'loading-dots'
}
</script>

<style lang="scss" scoped>

.loading-dots span {
  animation-name: blink;
  animation-duration: 1.5s;
  animation-iteration-count: infinite;
  animation-fill-mode: both;

  &:nth-child(2) {
    animation-delay: .5s;
  }

  &:nth-child(3) {
    animation-delay: 1s;
  }
}

@keyframes blink {
  0% {
    opacity: .2;
  }
  20% {
    opacity: 1;
  }
  100% {
    opacity: .2;
  }
}

</style>
