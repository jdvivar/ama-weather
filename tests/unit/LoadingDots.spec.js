import { shallowMount } from '@vue/test-utils'
import LoadingDots from '@/components/LoadingDots'

describe('LoadingDots', () => {
  it('has correct markup', () => {
    const wrapper = shallowMount(LoadingDots)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
