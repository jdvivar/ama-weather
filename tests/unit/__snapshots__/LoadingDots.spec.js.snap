// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingDots has correct markup 1`] = `<span class="loading-dots"><span>.</span><span>.</span><span>.</span></span>`;
